### ISSUE TITLE
 SUGGESTION ON RAINDROP BLUETOOTH OPTION #251 

### ISSUE THREAD

**Issue by kberey** [HydroBlockchain/Community-Brainstorming#40]
What is your suggestion?
Hi HydroTeam,

I would like to ask if there's an option if they're using a desktop browser with an internet connection but their phone has no WIFI or connection, How they can sync their Raindrops CODES?

I suggest if you have to put a 2nd option like bluetooth with password to connect the offline phone to desktop browser.

or can we use desktop to sync their hydro raindrop instead of using Phone?

Please advise.
Telegram: @kenberey

**Comment by bartsimpson1000** [comment]
I don't think this will work, because bluetooth requires one device to talk to another, and there is no second device. The community would have to create some kind of physical token or reader I think.
